### PR TITLE
feat(ui): Add the ability to name a scraper target

### LINKED ISSUE #11784: add the ability to name a scraper in the scraper configuration

### PR DESCRIPTION
**Pull request by ischolten**
Closes #11784 

_Briefly describe your proposed changes:_
Provides input for the scraper target to be named in scraper configuration
Update the client library dependency so get the correct Scraper type.

  - [x] Rebased/mergeable
  - [ ] Tests pass
  - [x] http/swagger.yml updated (if modified Go structs or API)
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ui/src/dataLoaders/components/configureStep/ConfigureDataSourceSwitcher.tsx [hunk 1]
```diff
@@ -31,8 +31,6 @@ class ConfigureDataSourceSwitcher extends PureComponent<Props> {
             <Scraping onClickNext={onClickNext} buckets={buckets} />
           </div>
         )
-      case DataLoaderType.CSV:
```
Comment (chnn) [inline ui/src/dataLoaders/components/configureStep/ConfigureDataSourceSwitcher.tsx]: rip

File: ui/src/dataLoaders/components/configureStep/ScraperTarget.tsx [hunk 2]
```diff
@@ -28,11 +30,21 @@ export class ScraperTarget extends PureComponent<Props> {
...
 
   public render() {
-    const {onSelectBucket, url, bucket, buckets} = this.props
+    const {onSelectBucket, url, name, bucket, buckets} = this.props
     return (
       <Grid>
         <Grid.Row>
           <Grid.Column widthXS={Columns.Eight} offsetXS={Columns.Two}>
+            <Form.Element label="Name">
+              <Input
```
Comment (chnn) [inline ui/src/dataLoaders/components/configureStep/ScraperTarget.tsx]: is a name allowed to be empty?

File: ui/src/dataLoaders/components/configureStep/ConfigureDataSourceSwitcher.tsx (hunk 1 above)
Comment (ischolten) [reply ui/src/dataLoaders/components/configureStep/ConfigureDataSourceSwitcher.tsx]: i cant believe that was in there for so long. oops

File: ui/src/dataLoaders/components/configureStep/ScraperTarget.tsx (hunk 2 above)
Comment (ischolten) [reply ui/src/dataLoaders/components/configureStep/ScraperTarget.tsx]: the api does not complain if name is empty.  i could add a check for it and give it a default name (Name this Scraper target) like we do with telegraf configs.

Comment (chnn) [reply ui/src/dataLoaders/components/configureStep/ScraperTarget.tsx]: a default or as-is seems good to me

Comment (ischolten) [reply ui/src/dataLoaders/components/configureStep/ScraperTarget.tsx]: i have added a default. it works the same as the scraper url now.

**Pull request merged**